CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and is generated by [Changie](https://github.com/miniscruff/changie).
 ### Added
 - Publish pre-built Linux ARM binaries.
 - {branch, stack, upstack, downstack} submit: Add --force flag. This acts like 'git push --force'.
+- Short flag for gs branch submit --file (added -f)
```

Comment:
This repo uses [changie](https://changie.dev/) to track unreleased changes.
This file only contains things that have already been released.
(https://github.com/abhinav/git-spice/blob/main/CONTRIBUTING.md#making-contributions.)

Reply:
oh woops, let me fix this